internal/graphql/resolver/schema.resolvers.go

```diff
@@ -108,6 +108,10 @@ func (r *queryResolver) FetchPatterns(ctx context.Context, selector model.PageFi
 	return r.fetchPatterns(ctx, provider, selector)
 }
 
+func (r *queryResolver) FetchFilters(ctx context.Context, selector model.PageFilter) (*model.FilterPage, error) {
+	panic(fmt.Errorf("not implemented"))
```

Comment:
panic... It will probably stop the meshery-server.

Reply:
@Abhishek-kumar09 these changes here are not required as it's covered in https://github.com/meshery/meshery/pull/6028/files#diff-b023c72e9cbb0937612593b7b30b8fcb9d965ec9c40a7e09fe3e5c432154ebb4

Reply:
@Chadha93 Those query does not include the `page`, `page_size` and `total_count` , which are used in the `configurations` pages .

Reply:
@Abhishek-kumar09 This was auto generated .